src/components/interface/pongshmem/doc.md

```diff
@@ -0,0 +1,9 @@
+## pongshmem
+
+This is the skeleton interface used by the `mkinterface.sh` script to aid in the creation of a new interface.
```

Comment:
Add some documentation here for the purpose of this interface. Likely just a couple of sentences.